docs/help.rst

```diff
@@ -70,68 +75,95 @@ contact support@robusta.dev or go to `our slack channel <https://bit.ly/robusta-
     For more info see:
     https://stackoverflow.com/questions/52805115/certificate-verify-failed-unable-to-get-local-issuer-certificate
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Helm installation fails
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Problems when running ``helm install`` command or installing via GitOps.
+
 .. details:: unknown field in com.coreos.monitoring.v1.Prometheus.spec, ValidationError(Prometheus.spec)
 
     This indicates potential discrepancies between the version of Prometheus you are trying to use and the version of the CRDs in your cluster.
 
     Follow this guide for :ref:`upgrading CRDs from an older version <Manual Upgrade>`.
 
-.. details:: robusta-runner isn't working or has exceptions
+.. details:: at least one sinks must be defined
+
+   Verify ``sinksConfig`` is defined in your Robusta values file, with at least one sink like Slack, Teams or Robusta UI ("robusta_sink"). If it's your first time installing, the fastest solution is to start configue creation from scratch.
+
+   .. code-block:: bash
+
+      Error: UPGRADE FAILED: execution error at (robusta/templates/playbooks-config.yaml:9:7): At least one sink must be defined!
 
-    Check the pod's memory consumption. If necessary, increase the memory request in the Helm values:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Robusta runner, Prometheus or Holmes failures
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. details:: robusta-runner pod is in Pending state due to memory issues
+
+    If your cluster has 20 Nodes or less, set robusta-runner's memory request to 512MiB in Robusta's Helm values:
 
     .. code-block:: yaml
 
         runner:
           resources:
             requests:
-              memory: 2048Mi
+              memory: 512MiB
             limits:
-              memory: 2048Mi
+              memory: 512MiB
 
-    Here's a representative error caused by too little memory:
+.. details:: Prometheus' pods are in Pending state due to memory issues
 
-    .. details:: Discovery Error
+        If your cluster has 20 Nodes or less, set Prometheus memory request to TBD
```

Comment:
To 1Gi (instead of 2Gi which is the default)
```
kube-prometheus-stack:
  prometheus:
    prometheusSpec:
      resources:
        requests:
          memory: 1Gi
        limits:
          memory: 1Gi

```